README.md

```diff
@@ -3,7 +3,7 @@
 Contributors: lukecarbis, ryankienstra, Stino11, rheinardkorf
 Tags: Gutenberg, Blocks
 Requires at least: 5.0
-Tested up to: 5.2
+Tested up to: 5.3
```

Comment:
It should be fine to bump this to `5.3.` whenever. The AMP plugin deployed last week [with this value](https://github.com/ampproject/amp-wp/blob/394b2023aaf7bb83146c3968d5b43e8be194cac7/readme.txt#L5), and its 'Tested up to' on wp.org [is 5.3](https://wordpress.org/plugins/amp/).